core/API/DataTableManipulator/LabelFilter.php

```diff
@@ -102,6 +106,9 @@ private function doFilterRecursiveDescend($labelParts, $dataTable)
     protected function manipulateSubtableRequest($request)
     {
         unset($request['label']);
+        unset($request['flat']);
+        $request['totals'] = 0;
```

Comment:
This makes the label filter much faster. For each requested subtable it did before calculate the "report totals" and therefore for each subtable request the 1st level data table in another request